src/components/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
+import { ExclamationCircleIcon } from '@heroicons/react/24/solid'
+
+/**
+ * Creates an alert with the given string message.
+ * 
+ * @returns The alert component
+ */
+const Alert = (props: {message: string, className?: string}): ReactElement =>  {
+    return (
+        <div className={`flex items-start space-x-2 text-left bg-[#FFEDC0] rounded-lg p-3 text-sm ${props.className}`} role="alert">
```

Comment:
I'd recommend putting the color hex into some kind of variable so it's easier to update in the future if needed, rather than having to directly edit this inline style.

Reply:
Especially since colors can be a bit more flexible than other CSS styles

Reply:
Definitely. Would we do this as a variable in TS or just use CSS?

Reply:
I'd check this out: https://tailwindcss.com/docs/reusing-styles

And utilize the `@apply`

Reply:
Updated!